Moderation-Policy.md

```diff
@@ -237,6 +238,22 @@ repository by repeatedly sharing or discussing the details of nodejs/moderation
 issues in any public forum or social media service risks being permanently
 removed from the Node.js GitHub organization.
 
+## Privacy of the nodejs/collaborators Team Discussion Board
+
+The nodejs/collaborators team discussion board is sometimes used for discussing 
+things that don't directly fit the nodejs/node repository. For example nominations
+of Contributors for Collaborator status might be discussed there in order to give
+existing Collaborators a chance to voice their opinion on the nomination.
+
+The nodejs/collaborators team discussion board is considered private but accessible to
+all Collaborators. The details of any issue discussed within it are expected to remain
+confidential and are not to be discussed in any public forum or social media service.
+
+Any Collaborator found to be violating the privacy of it by repeatedly sharing or
```

Comment:
`privacy of it` is a bit awkward here, although I see it mirrors `privacy of the nodejs/moderation repository` in the previous section. In both places, I would change `privacy` to `confidentiality`. And here, I would change `of it` to `of the nodejs/collaborators Team Discussion Board`.